shoes-core/lib/shoes/dimensions.rb

```diff
@@ -82,6 +82,16 @@ def in_bounds?(x, y)
       x_dimension.in_bounds?(x) && y_dimension.in_bounds?(y)
     end
 
+    def contains?(other)
+      return false unless other.element_left && other.element_top &&
+                          other.element_right && other.element_bottom
+
+      element_left <= other.element_left &&
+        element_right >= other.element_right &&
+        element_top <= other.element_top &&
+        element_bottom >= other.element_bottom
```

Comment:
partial contains aren't a it?

Reply:
I found that showing on partial containment looked way jankier than just hiding as soon as things got outside the box.

Should the method have a more precise name than `contains?`, like `fully_contains?`?